gitlab/utils.py

```diff
@@ -55,7 +55,11 @@ def response_content(
 
 
 def _transform_types(
-    data: Dict[str, Any], custom_types: dict, *, transform_files: Optional[bool] = True
+    data: Dict[str, Any],
+    custom_types: dict,
+    *,
+    is_params: bool,
```

Comment:
Would it make sense to have this as `transform_data` since we already have `data` in there? Just a thought for a follow-up, not needed now. Then it gives us a nice `if not transform_data and not transform_files:`.

It'd maybe also be cool to have this with a default value just like `transform_files` so we only pass it around when we have to override. But also non-blocking.

Reply:
> Would it make sense to have this as `transform_data` since we already have `data` in there? Just a thought for a follow-up, not needed now. Then it gives us a nice `if not transform_data and not transform_files:`.

Sure.

> It'd maybe also be cool to have this with a default value just like `transform_files` so we only pass it around when we have to override. But also non-blocking.

I don't like the idea of a default value. IMHO, default values are for things whose value will be the default for the large majority of the cases. I don't feel like this one is that case.